lib/fields/utils/wrapped.js

```diff
@@ -28,6 +28,7 @@ var makeTitle = function(description, errors) {
 var FieldWrapper = React.createClass({
   render: function() {
     var classes = [].concat(errorClass(this.props.errors) || [],
+                            'form-group',
```

Comment:
FieldWrapper is already an element that you can pass to Form in order to customize the html layout. You don't need this value here.
The preferred way to achieve the same goal is by creating a custom FieldWrapper in nutella and pass it to Form. Same thing for SectionWrapper.

https://github.com/mosaicjs/mosaic-ui-form/blob/master/lib/FormView.js

Look up for BootstrapFieldWrapper and BootstrapSectionWrapper. The render methods contain what we are looking for.